mqt/qfr/qiskit/QuantumCircuit.hpp

```diff
@@ -192,6 +197,77 @@ namespace qc::qiskit {
             }
         }
 
+        static SymbolOrNumber parseSymbolicExpr(const py::object& pyExpr) {
+            static const std::regex summands("[+|-]?[^+-]+");
+            static const std::regex products("[\\*/]?[^\\*/]+");
+            // static const std::regex products(".*");
+            // static const products("")
```

Comment:
can these left-over comments be removed?

Reply:
Removed them.